app/models/issuer.py

```diff
@@ -98,3 +102,39 @@ class ClearPendingRevocationsResult(BaseModel):
             "The resulting revocations that are still pending after a clear-pending request has been completed."
         ),
     )
+
+
+class RevokedResponse(BaseModel):
+    cred_rev_ids_published: Dict[str, List[int]] = Field(
+        default_factory=dict,
+        description=(
+            "A map of revocation registry IDs to lists of credential revocation IDs "
+            "(as integers) that have been revoked. Can be empty."
```

Comment:
"Can be empty if ..." We should say what an empty list means. In this context, an empty list of a cred_rev_ids means no creds were revoked for that rev_reg_id?